.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.idea
```

Comment:
I think it is possible to set a global git ignore value for all the git repositories on your development machine. Maybe that would be a good solution?